Tests/scripts/update_id_set.py

```diff
@@ -242,7 +242,7 @@ def get_script_data(file_path, script_code=None):
     deprecated = data_dictionary.get('deprecated')
     fromversion = data_dictionary.get('fromversion')
     depends_on, command_to_integration = get_depends_on(data_dictionary)
-    script_executions = sorted(list(set(re.findall("demisto.executeCommand\(['\"](\w+)['\"].*", script_code))))
+    script_executions = sorted(list(set(re.findall("demisto.executeCommand\\(['\"](\w+)['\"].*", script_code))))
```

Comment:
Not sure why you added this `\` we don't want to identify this char, but strangely enough it catches the same thing on my env

Reply:
yes it is the same. only clearer (at least for me) if not using a raw string. we are basically passing the \( to the regex. we can leave as is.